doc/install-on-docker.md

```diff
@@ -1,10 +1,53 @@
 # Installation on Docker
 
-You can install Ray on any platform that runs Docker. We do not presently publish Docker images for Ray, but you can build them yourself using the Ray distribution. Using Docker can provide a reliable way to get up and running quickly.
+You can install Ray on any platform that runs Docker. We do not presently publish Docker images for Ray, but you can build them yourself using the Ray distribution.
+
+Using Docker can streamline the build process reliable way to get up and running quickly.
 
 ## Install Docker
 
-The Docker Platform release is available for Mac, Windows, and Linux platforms. Please download the appropriate version from the [Docker website](https://www.docker.com/products/overview#/install_the_platform).
+### Mac, Linux, Windows platforms
+
+The Docker Platform release is available for Mac, Windows, and Linux platforms. Please download the appropriate version from the [Docker website](https://www.docker.com/products/overview#/install_the_platform) and follow the corresponding installation instructions.
+
+### Docker installation on EC2 with Ubuntu
```

Comment:
The official Linux instructions seem to focus on connecting to Dockerhub and getting a repository set up. These are simple instructions to get Docker installed and to get it to run without sudo.

The `dist-upgrade` command here is likely optional. It' something I always do just so I'm running the latest patched version of the OS. I know things have at times broken without it but I don't remember whether it is necessary right now. I think it's probably safest to leave it in here.